src/ips/actions.ts

```diff
@@ -10,7 +10,9 @@ export enum Actions {
   PUT = 'http://schemas.xmlsoap.org/ws/2004/09/transfer/Put',
   DELETE = 'http://schemas.xmlsoap.org/ws/2004/09/transfer/Delete',
   SETUP = 'http://intel.com/wbem/wscim/1/ips-schema/1/IPS_HostBasedSetupService/Setup',
+  ADMIN_SETUP = 'http://intel.com/wbem/wscim/1/ips-schema/1/IPS_HostBasedSetupService/AdminSetupResponse',
+  ADD_NEXT_CERT_IN_CHAIN = 'http://intel.com/wbem/wscim/1/ips-schema/1/IPS_HostBasedSetupService/AddNextCertInChainResponse',
   START_OPT_IN = 'http://intel.com/wbem/wscim/1/ips-schema/1/IPS_OptInService/StartOptIn',
   CANCEL_OPT_IN = 'http://intel.com/wbem/wscim/1/ips-schema/1/IPS_OptInService/CancelOptIn',
-  SEND_OPT_IN_CODE = 'http://intel.com/wbem/wscim/1/ips-schema/1/IPS_OptInService/SendOptInCode'
+  SEND_OPT_IN_CODE = 'http://intel.com/wbem/wscim/1/ips-schema/1/IPS_OptInService/SendOptInCode',
```

Comment:
should this last line have a comma?

Reply:
whoops golang likes that comma but I forgot that nobody else does